js/examples/openai/frontend/app/components/transform.ts

```diff
@@ -6,6 +6,30 @@ export const isValidMessageData = (rawData: JSONValue | undefined) => {
   return true;
 };
 
+const getSpanIdFromAnnotations = (annotations?: JSONValue[]) => {
+  if (!annotations) return;
+  for (const annotation of annotations) {
+    if (
+      annotation != null &&
+      typeof annotation === "object" &&
+      "spanId" in annotation &&
+      typeof annotation.spanId === "string"
+    ) {
```

Comment:
next build didn't accept a typeguard here

Reply:
lol yeah i don't know how they are compiling but i pulled this out into a typeguard function and the build process failed saying spanId is not a property of string...